### DISCUSSION TITLE
support bookmarks inside system collections (files and users)

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#5997] [Feature Requests]
with the ability to create and recall bookmarks like it's already working for the user collections

**Comment by azrikahar** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
